dev/doc/build-system.txt

```diff
@@ -143,6 +143,8 @@ file list(s):
    These files are also used by the experimental ocamlbuild plugin,
    which is quite touchy about them : be careful with order,
    duplicated entries, whitespace errors, and do not mention .mli there.
+   If module B depends on module A, then B should be after A in the .mllib
+   file.
```

Comment:
Since this is unrelated, it would be great if it could be in a separate commit, even if kept in the same PR to avoid confusion when people try to make sense of this commit in the future.

Reply:
Done.